car.capnp

```diff
@@ -66,6 +66,7 @@ struct CarEvent @0x9b1657f34caf3ad3 {
     lowBattery @48;
     vehicleModelInvalid @50;
     accFaulted @51;
+    gmAccFaultedTemp @115;
```

Comment:
We can rename to accFaultedTemp if we end up using this for other makes

Reply:
This would be useful for VW, we explicitly have both temp and perm faults.

Reply:
Cool, open a PR to rename it when you want to use it. In GM's case it's to prevent an ACC fault from happening in certain situations